library/src/main/java/com/google/android/exoplayer2/text/Cue.java

```diff
@@ -134,10 +143,26 @@
   /**
    * The size of the cue box in the writing direction specified as a fraction of the viewport size
    * in that direction, or {@link #DIMEN_UNSET}.
+   * <p>
+   * If {@link #bitmap} is not null then this value is used to indicate the width
    */
   public final float size;
 
   /**
+   * The height size of the cue box when a {@link #bitmap} is set specified as a fraction of the
+   * viewport size in that direction, or {@link #DIMEN_UNSET}.
+   */
+  public final float size_height;
```

Comment:
`size` should just be the fraction of the video's width occupied by the bitmap. `position` and `line` should be the fractional position of the respective anchors. I don't think you need `size_height` at all.